azure-pipelines.yml

```diff
@@ -36,7 +36,7 @@ jobs:
         swig3.0 \
         libpython2.7-dev \
         libzmq5 libzmq3-dev \
-        libboost-all-dev
+        libboost1.71-dev
```

Comment:
>libboost1.71-dev [](start = 8, length = 16)

I see it already in slave container: sonic-slave-buster/Dockerfile.j2:324
Will you just remove `libboost-all-dev` ?

Reply:
I did not notice this docker is deriving from slave buster. I was under the impression it is a parallel one. Will remove